src/content/models.py

```diff
@@ -366,6 +366,14 @@ def get_first_publisher(self) -> Optional[UserModel]:
 
         return None
 
+    def get_latest_publisher(self) -> Optional[UserModel]:
+        """Return the latest publisher of the page or None."""
+        latest_revision_with_user = (
+            self.revisions.exclude(user=None).order_by("created_at", "id").last()
+        )
+
+        return latest_revision_with_user.user if latest_revision_with_user else None
```

Comment:
```suggestion
        if latest_revision_with_user := (
            self.revisions.exclude(user=None).order_by("created_at", "id").last()
        ):
            return latest_revision_with_user.user  
        return None
```
what you have is fine, maybe this is nicer?

Reply:
Sorted this and refactored the method above for consistency.